server/cli/cli.go

```diff
@@ -43,6 +43,9 @@ Output build version information.
 		fmt.Fprintf(w, "Build Time:  %s\n", info.Time)
 		fmt.Fprintf(w, "Build Deps:\n\t%s\n",
 			strings.Replace(strings.Replace(info.Deps, " ", "\n\t", -1), ":", "\t", -1))
+		if err := w.Flush(); err != nil {
+			fmt.Println(err)
```

Comment:
it's an unknown error for all we know, so either we handle it or we don't. Seems predestined for a panic if you ask me.